### ISSUE TITLE
Scroll jumps on docs

### ISSUE THREAD

**Issue by eridal** [vulcand/vulcand#128]
Browser: latest Firefox mobile

Whenever you scroll, the page starts moving down and, when it's finally slowing, the scroll jumps up some lines, making really hard to follow the reading. You're continuously forced to re-scroll down a little bit extra, which sometimes causes the bug again..

**Comment by klizhentas** [comment]
cc @leemunroe hey Lee, Vulcand docs use Mailgun docs, is it something you can take a look at?
